### ISSUE TITLE
`download` icon instead `media-play`

### ISSUE THREAD

**Issue by artemanufrij** [cryptowyrm/copypastegrab#3]
use `emblem-downloads` icon instead `media-play`
![screenshot von 2018-11-18 12 21 27](https://user-images.githubusercontent.com/16517095/48671711-c7531780-eb2c-11e8-9ff7-ad1543fc086b.png)

**Comment by cryptowyrm** [comment]
Hi, thanks for your suggestion! :)

I'm not sure about this change. Using play/stop icons for downloads is the most intuitive I believe because it's the standard used in download managers, it's more consistent with the use of the stop icon to pause/cancel downloads and also hints at the fact that there is a way to pause downloads, so it carries a lot more information than emblem-downloads.

I tried the change out to see how it looks:

![screenshot from 2018-11-18 14 40 47](https://user-images.githubusercontent.com/16908445/48673359-b57b6f80-eb40-11e8-9390-5e55af25f76a.png)

I'm not convinced this is an improvement but I could be completely wrong, so I'll wait a bit for more feedback on this. I'm happy to bend to the will of the users here :)

**So to anyone else who reads this, please give @artemanufrij's post at the top a thumbs up if you also want this change.**

**Comment by artemanufrij** [comment]
I agree with you about stop/pause icons. but if I click a play button I expect to play a video and not download it!

**Comment by cryptowyrm** [comment]
Yes, since it's videos that are downloaded it could be confusing. I'll keep the change in a branch so that I can quickly merge it before I release the next version and we've hopefully gotten a bit more feedback on this from other users. I associate the emblem-downloads icon mostly with the downloads folder icon so I find it's use here a bit out of place, but it might just be me :)
